### PR TITLE
Verify JavaDoc and Code Cleanup

### PR DESCRIPTION
**Pull request by JawadHossain**
=> replace hard coded strings
=> update comments

### CODE REVIEW

File: app/src/androidTest/java/com/example/bookwormadventuresdeluxe2/MyBooksFragmentTest.java [hunk 1]
```diff
@@ -43,7 +43,7 @@
     private Resources resources;
 
     private int BACK_BUTTON_INDEX = 0;
-    /* In the edit book fragment the index is different because android is stupid and picks
+    /* In the edit book fragment the index is different as android picks
```
Comment (KateVlaar) [inline app/src/androidTest/java/com/example/bookwormadventuresdeluxe2/MyBooksFragmentTest.java:46]: Lol nice catch

**Review by KateVlaar — APPROVED** [review]
This is much appreciated, thank you

File: app/src/androidTest/java/com/example/bookwormadventuresdeluxe2/MyBooksFragmentTest.java (hunk 1 above)
Comment (huddy987) [reply app/src/androidTest/java/com/example/bookwormadventuresdeluxe2/MyBooksFragmentTest.java:46]: this was my comment I was getting mad when writing that test lol

**Review by huddy987 — APPROVED** [review]

File: app/src/androidTest/java/com/example/bookwormadventuresdeluxe2/MyBooksFragmentTest.java (hunk 1 above)
Comment (JawadHossain) [reply app/src/androidTest/java/com/example/bookwormadventuresdeluxe2/MyBooksFragmentTest.java:46]: Haha 😂

**Pull request merged**